frontend/src/container/RoutingPolicies/RoutingPolicyList.tsx

```diff
@@ -0,0 +1,73 @@
+import { Table, TableProps, Typography } from 'antd';
+import { useMemo } from 'react';
+
+import RoutingPolicyListItem from './RoutingPolicyListItem';
+import { RoutingPolicy, RoutingPolicyListProps } from './types';
+
+function RoutingPolicyList({
+	routingPolicies,
+	isRoutingPoliciesLoading,
+	isRoutingPoliciesError,
+	handlePolicyDetailsModalOpen,
+	handleDeleteModalOpen,
+}: RoutingPolicyListProps): JSX.Element {
+	const columns: TableProps<RoutingPolicy>['columns'] = [
+		{
+			title: 'Routing Policy',
+			key: 'routingPolicy',
+			render: (data: RoutingPolicy): JSX.Element => (
+				<RoutingPolicyListItem
+					routingPolicy={data}
+					handlePolicyDetailsModalOpen={handlePolicyDetailsModalOpen}
+					handleDeleteModalOpen={handleDeleteModalOpen}
+				/>
+			),
+		},
+	];
+
+	const localeEmptyState = useMemo(
+		() => (
+			<div className="no-routing-policies-message-container">
+				{isRoutingPoliciesError ? (
+					<img src="/Icons/awwSnap.svg" alt="aww-snap" className="error-state-svg" />
+				) : (
+					<img
+						src="/Icons/emptyState.svg"
+						alt="thinking-emoji"
+						className="empty-state-svg"
+					/>
+				)}
+				{isRoutingPoliciesError ? (
+					<Typography.Text>
+						Something went wrong while fetching routing policies.
```

Comment:
Not in this PR, but when there is an error state, there should be a way to retry with some CTA, without that the only action user can do is page reload which can be little frustrating.